### DISCUSSION TITLE
Is my AdGuard Home version (0.107.7) outdated and causing performance issues on QNAP?

### DISCUSSION THREAD

**Discussion by FFernandoCS** [AdguardTeam/AdGuardHome#7328] [Q&A]
Hi,

I’ve been experiencing slower DNS responses and frequent CPU spikes on my QNAP, caused by AdGuard Home. The issue started about three weeks ago, even though I haven’t changed anything.

I'm concerned that I might be running version 0.107.7, and I’m not sure if it’s outdated or if something went wrong during installation (possibly due to a localization patch). I initially installed AdGuard Home in 2022 and applied updates whenever available, but I can't recall the last update date.

Could anyone help verify if this version might be the cause of these issues?

Thanks!
![adguardhomever](https://github.com/user-attachments/assets/781209fe-c051-4449-97bf-48427ff517cf)

**Comment by Dynamic5912** [comment]
Have you tried updating to the latest version to see if it fixes the issue?
